pkg/util/logging/default_logger.go

```diff
@@ -33,9 +33,13 @@ func DefaultHooks() []logrus.Hook {
 
 // DefaultLogger returns a Logger with the default properties
 // and hooks.
-func DefaultLogger(level logrus.Level) *logrus.Logger {
+func DefaultLogger(level logrus.Level, j bool) *logrus.Logger {
```

Comment:
I would probably rename this new variable `json`, and extend the godoc.

Just reading the function signature, it's not clear what `j` means.

Reply:
Thanks for the feedback and taking a look... makes sense, I've pushed some updates